core/src/main/java/org/apache/carbondata/core/datastore/columnar/BlockIndexerStorageForInt.java

```diff
@@ -27,25 +27,26 @@
 public class BlockIndexerStorageForInt implements IndexStorage<int[]> {
   private boolean alreadySorted;
 
-  private int[] dataAfterComp;
+  private int[] rowIdPage;
 
-  private int[] indexMap;
+  private int[] rowIdRlePage;
 
-  private byte[][] keyBlock;
+  private byte[][] dataPage;
 
-  private int[] dataIndexMap;
+  private int[] dataRlePage;
 
   private int totalSize;
 
-  public BlockIndexerStorageForInt(byte[][] keyBlock, boolean compressData, boolean isNoDictionary,
+  public BlockIndexerStorageForInt(byte[][] dataPage, boolean rleOnData, boolean isNoDictionary,
       boolean isSortRequired) {
-    ColumnWithIntIndex[] columnWithIndexs = createColumnWithIndexArray(keyBlock, isNoDictionary);
+    ColumnWithIntIndex[] dataWithRowId = createColumnWithIndexArray(dataPage, isNoDictionary);
```

Comment:
suggest to change as following.
ColumnWithIntIndex => ColumnWithRowId
createColumnWithIndexArray => createColumnWithRowId